client/src/docs-ui/internal-link/internal-link.tsx

```diff
@@ -22,6 +22,8 @@ export class DocsInternalLink {
   @Prop() readonly activeClass?: string;
   /*** class name to attach a subpage is active */
   @Prop() readonly childActiveClass?: string;
+  /*** override `isChildActive` to true */
+  @Prop() readonly overrideChildActiveToTrue?: boolean;
```

Comment:
How does this act as an override in a readonly state?

Reply:
This prop was removed